### PR TITLE
Finish mobile version

### PR DESCRIPTION
**Pull request by sagieramos**
### New Features
- full mobile page
- added form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,57 +9,246 @@
...
+                        <h3>Framework</h3>
+                        <ion-icon class="ionicon" name="chevron-forward-outline"></ion-icon>
+                    </div>
+                    <div class="h3-container box">
+                        <h3>Skills</h3>
+                        <ion-icon class="ionicon" name="chevron-forward-outline"></ion-icon>
+                    </div>
+                </div>
+            </section>
         </section>
```
Comment (Qoosim) [inline index.html:234]: - [ ] It is required that the technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li. Kindly change that too.
- [ ] It is equally required that you must produce a close replica of the Figma design. Your design look differently to how it is on the Figma design. Please, take a look.

### Your Design
your-lang

### Figma Design
temp-lang

File: index.html [hunk 2]
```diff
@@ -9,57 +9,246 @@
...
+                        </li>
+                        <li>
+                            <a href="#"><img src="./assets/social/github.svg" alt="Github"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./assets/social/angellist.svg" alt="AngelList"></a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="btn"><a href="#">Get resume</a></div>
```
Comment (Qoosim) [inline index.html:201]: - [ ] Kindly complete the text inside the div to match what is on the Figma design

### Your Design
your-btn

### Figma Design
temp-btn

File: index.html [hunk 3]
```diff
@@ -9,57 +9,246 @@
...
+                        <li>
+                            <a href="#"><img src="./assets/social/medium.svg" alt="Meduim"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./assets/social/github.svg" alt="Github"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./assets/social/angellist.svg" alt="AngelList"></a>
+                        </li>
+                    </ul>
```
Comment (Qoosim) [inline index.html:199]: - [ ] Kindly pay attention to the little margin left to the list of the icons on the Figma design and apply the same on your design

### Your Design
your-icon-space

### Figma Design
temp-icon-space

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @sagieramos, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by NahnahAJ — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @sagieramos, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- All requested changes implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**